README.md

```diff
@@ -18,16 +16,11 @@ development process and policies.
 * Build it using Dockerfile
 
     ```ssh
-    $ git clone https://github.com/dixudx/trac-docker
+    $ git clone https://github.com/ttkk1024/trac-docker
     $ cd trac-docker
     $ docker build -t trac ./
     ```
 
-* just pull it from Dockerhub
```

Comment:
I think we could add a github action to automatically build and upload container images to Dockerhub.